docs/repositories.rst

```diff
@@ -0,0 +1,47 @@
+Repositories
+============
+
+mozilla-central
+---------------
+
+Browser code is maintained in the mozilla-central. This includes several areas, related to addons. All of the following bug trackers are in Bugzilla:
```

Comment:
> Browser code is maintained in the mozilla-central.

This shouldn't have the word `the` in it or it should the be the `repository` or somthing.